src/OpenTelemetry/Metrics/Aggregators/Aggregator.cs

```diff
@@ -47,5 +61,23 @@ public abstract class Aggregator<T>
         /// </summary>
         /// <returns><see cref="AggregationType"/>.</returns>
         public abstract AggregationType GetAggregationType();
+
+        /// <summary>
+        /// Determines the last end timestamp before the current interval's <see cref="startTime"/>.
+        /// </summary>
+        /// <returns>The end timestamp of the last aggregated checkpoint.</returns>
+        protected DateTime GetLastEndTimestamp()
```

Comment:
It's better use `DateTimeOffset` in such APIs. DateTime is designed for UI-like interactions

Reply:
```suggestion
        protected DateTimeOffset GetLastEndTimestamp()
```

Reply:
I am making this change, but just FYI, the MetricData.Timestamp exported is a DateTime. So this means I have to convert in each of the aggregators to DateTime. I suppose we could change the Timestamp in MetricData to be DateTime, but I don't know if that's something we'd want to do, since it would be changing external facing stuff. https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/src/OpenTelemetry/Metrics/Export/MetricData.cs#L27

Reply:
Done